### ISSUE TITLE
[FE] Breadcrumb angle bracket is read aloud by Voice Over

### ISSUE THREAD

**Issue by wullaski** [department-of-veterans-affairs/va.gov-team#53066]
You can select it as readable text and when the page loads it is read aloud after the H1 is read. Ex: "Breadcrumb, single left pointing quotation mark, back to appointments"

**Comment by brianseek** [comment]
@wullaski can you validate this one since you were hearing the bracket being read aloud? I just merged it and should be out on stage shortly

**Comment by wullaski** [comment]
@brianseek validated on staging. Sounds good 🔈
